e2e-tests/playwright/support/ui/components/channels/account_settings/account_settings_modal.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {expect, Locator} from '@playwright/test';
+
+import {NotificationsSettings} from './notification_settings';
+
+export default class AccountSettingsModal {
+    readonly container: Locator;
+
+    readonly notificationsSettingsTab;
+    readonly notificationsSettings;
+
+    constructor(container: Locator) {
+        this.container = container;
+
+        this.notificationsSettingsTab = this.container.locator('#notificationsButton');
+        this.notificationsSettings = new NotificationsSettings(container.locator('#notificationSettings'));
+    }
+
+    async toBeVisible() {
+        await expect(this.container).toBeVisible();
```

Comment:
@saturninoabril The test keep failing here even though the setting modal opens up. Can you please try to run locally and check how i can improve here